src/Microsoft.Data.SqlClient/src/Microsoft/Data/SqlClient/SqlClientEventSource.cs

```diff
@@ -924,148 +893,111 @@ internal void EndExecute(int objectId, int compositeState, int sqlExceptionNumbe
         }
 
         [Event(AdvancedScopeExitId, Level = EventLevel.Verbose, Opcode = EventOpcode.Stop, Keywords = Keywords.AdvancedTrace)]
-        internal void AdvancedScopeLeave(long scopeId)
-        {
-            WriteEvent(AdvancedScopeExitId, scopeId);
-        }
+        internal void AdvancedScopeLeave(string message) =>
+            WriteEvent(AdvancedScopeExitId, message);
 
         [Event(AdvancedTraceBinId, Level = EventLevel.Verbose, Keywords = Keywords.AdvancedTraceBin)]
-        internal void AdvancedTraceBin(string message)
-        {
+        internal void AdvancedTraceBin(string message) =>
             WriteEvent(AdvancedTraceBinId, message);
-        }
 
         [Event(AdvancedTraceErrorId, Level = EventLevel.Error, Keywords = Keywords.AdvancedTrace)]
-        internal void AdvancedTraceError(string message)
-        {
+        internal void AdvancedTraceError(string message) =>
             WriteEvent(AdvancedTraceErrorId, message);
-        }
 
         [Event(CorrelationTraceId, Level = EventLevel.Informational, Keywords = Keywords.CorrelationTrace, Opcode = EventOpcode.Start)]
-        internal void CorrelationTrace(string message)
-        {
+        internal void CorrelationTrace(string message) =>
             WriteEvent(CorrelationTraceId, message);
-        }
 
         [Event(StateDumpEventId, Level = EventLevel.Verbose, Keywords = Keywords.StateDump)]
-        internal void StateDump(string message)
-        {
+        internal void StateDump(string message) =>
             WriteEvent(StateDumpEventId, message);
-        }
 
         [Event(SNITraceEventId, Level = EventLevel.Informational, Keywords = Keywords.SNITrace)]
-        internal void SNITrace(string message)
-        {
+        internal void SNITrace(string message) =>
             WriteEvent(SNITraceEventId, message);
-        }
 
         [Event(SNIScopeEnterId, Level = EventLevel.Informational, Opcode = EventOpcode.Start, Keywords = Keywords.SNIScope)]
         internal long SNIScopeEnter(string message)
         {
             long scopeId = Interlocked.Increment(ref s_nextSNIScopeId);
-            WriteEvent(SNIScopeEnterId, message);
+            WriteEvent(SNIScopeEnterId, string.Format(message, scopeId));
             return scopeId;
         }
 
         [Event(SNIScopeExitId, Level = EventLevel.Informational, Opcode = EventOpcode.Stop, Keywords = Keywords.SNIScope)]
-        internal void SNIScopeLeave(long scopeId)
-        {
-            WriteEvent(SNIScopeExitId, scopeId);
-        }
+        internal void SNIScopeLeave(string message) =>
+            WriteEvent(SNIScopeExitId, message);
         #endregion
     }
 
     internal readonly struct SNIEventScope : IDisposable
     {
-        private readonly long _scopeID;
+        private readonly long scopeId;
 
-        public SNIEventScope(long scopeID)
-        {
-            _scopeID = scopeID;
-        }
+        public SNIEventScope(long scopeID) => scopeId = scopeID;
```

Comment:
NIT: Could you explain why did you remove the `_` from the `_scopeID` if you did it intentionally; otherwise, I suggest the following naming:
- use `_scopeId` instead of `scopeId` and `scopeId` instead of 'scopeID'.